tools/Session.php

```diff
@@ -31,6 +33,11 @@ public function setKey($key, $value){
 
 	}
 
+	public function setValueKey($key, $valueKey, $value){
```

Comment:
Codacy found an issue: [setValueKey accesses the super-global variable $_SESSION.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=5982252)